wordpress-mobile/WordPress-Android
# Page search crash fix

Fixes #9259.

To test:
1. Go To Site Pages
2. Tap on the Search toolbar button
3. Start typing to initiate searching
4. Notice the app does not crash and search results are returned

Note: The featured images are not displayed in search on purpose in order to prevent performance degradation when the results are updated frequently during typing

Update release notes:
